plugin-best-practices-plugin/src/functionalTest/groovy/com/autonomousapps/fixture/SimplePluginProject.groovy

```diff
@@ -228,6 +245,6 @@ final class SimplePluginProject {
pis","trace":{"trace":[{"owner":"com/test/GreetingPlugin","name":"baz","descriptor":"()V"},{"owner":"org/gradle/api/tasks/TaskContainer","name":"all","descriptor":"(Lorg/gradle/api/Action;)V"}]}},{"type":"com.autonomousapps.issue.GetAllprojectsIssue","name":"getAllprojects","trace":{"trace":[{"owner":"com/test/GreetingPlugin","name":"apply","descriptor":"(Ljava/lang/Object;)V"},{"owner":"com/test/GreetingPlugin","name":"apply","descriptor":"(Lorg/gradle/api/Project;)V"},{"owner":"org/gradle/api/Project","name":"getAllprojects","descriptor":"()Ljava/util/Set;"}]}},{"type":"com.autonomousapps.issue.GetProjectInTaskActionIssue","name":"getProject","trace":{"trace":[{"owner":"com/test/FancyTask","name":"action","descriptor":"()V","metadata":{"isTaskAction":true}},{"owner":"com/test/FancyTask","name":"doAction","descriptor":"()V"},{"owner":"com/test/FancyTask$ReallyFancyTask","name":"doAction","descriptor":"()V"},{"owner":"com/test/FancyTask$ReallyFancyTask","name":"getProject","descriptor":"()Lorg/gradle/api/Project;"}]}},{"type":"com.autonomousapps.issue.GetProjectInTaskActionIssue","name":"getProject","trace":{"trace":[{"owner":"com/test/ParentTask","name":"action","descriptor":"()V","metadata":{"isTaskAction":true}},{"owner":"com/test/ParentTask","name":"foo","descriptor":"()V"},{"owner":"com/test/ParentTask","name":"bar","descriptor":"()V"},{"owner":"com/test/ParentTask","name":"doAction","descriptor":"()V"},{"owner":"com/test/ParentTask$ChildTask","name":"doAction","descriptor":"()V"},{"owner":"com/test/ParentTask$ChildTask","name":"getProject","descriptor":"()Lorg/gradle/api/Project;"}]}},{"type":"com.autonomousapps.issue.GetProjectInTaskActionIssue","name":"getProject","trace":{"trace":[{"owner":"com/test/ParentTask2","name":"action","descriptor":"()V","metadata":{"isTaskAction":true}},{"owner":"com/test/ParentTask2","name":"doAction","descriptor":"()V"},{"owner":"com/test/ParentTask2$ChildTask2","name":"doAction","descriptor":"()V"},{"owner":"com/test/ParentTask2$ChildTask2","name":"foo","descriptor":"()V"},{"owner":"com/test/ParentTask2$ChildTask2","name":"bar","descriptor":"([ILjava/lang/String;)V"},{"owner":"com/test/ParentTask2$ChildTask2","name":"getProject","descriptor":"()Lorg/gradle/api/Project;"}]}},{"type":"com.autonomousapps.issue.GetSubprojectsIssue","name":"getSubprojects","trace":{"trace":[{"owner":"com/test/GreetingPlugin","name":"apply","descriptor":"(Ljava/lang/Object;)V"},{"owner":"com/test/GreetingPlugin","name":"apply","descriptor":"(Lorg/gradle/api/Project;)V"},{"owner":"org/gradle/api/Project","name":"getSubprojects","descriptor":"()Ljava/util/Set;"}]}},{"type":"com.autonomousapps.issue.SubprojectsIssue","name":"subprojects","trace":{"trace":[{"owner":"com/test/GreetingPlugin","name":"apply","descriptor":"(Ljava/lang/Object;)V"},{"owner":"com/test/GreetingPlugin","name":"apply","descriptor":"(Lorg/gradle/api/Project;)V"},{"owner":"org/gradle/api/Project","name":"subprojects","descriptor":"(Lorg/gradle/api/Action;)V"}]}}]
```

Comment:
I really should make this easier to maintain...

Reply:
Yea that took me some time to figure out 😅